deltametrics/cube.py

```diff
@@ -17,13 +19,15 @@
 from . import plot
 
 
-class CubeVariable(np.ndarray):
+@xr.register_dataarray_accessor("cubevar")
+class CubeVariable():
```

Comment:
I think some of the clunkiness can be taken out by adding a `__getitem__` to this class. Adding the following makes it so that you don't have to do `rcm8cube['eta'].data[10,:,:]`, but can just keep the straightforward API we have in place (i.e., `rcm8cube['eta'][10,:,:]`) and the object you get back is still a proper `xarray` `DataArray`. I think this will be something we definitely want to straighten out before merging. This should simplify a lot of additional `.data` calls.

```python
    def __getitem__(self, slc):
        return self.data[slc]
```